Source/Atomic/Math/StringHash.cpp

```diff
@@ -43,20 +43,24 @@ const StringHash StringHash::ZERO;
 StringHash::StringHash(const char* str) :
     value_(Calculate(str))
 {
+#if ATOMIC_PROFILING
+    RegisterSignificantString(str, *this);
+#endif

```

Comment:
We need this for being able to recognize event names. Events as you know are sent using `StringHash`, but profiler needs actual strings for data to be useful. See [here](https://github.com/rokups/AtomicGameEngine/blob/4b4cc5027f1fc15d8bb9ad4f488c91f687540d10/Source/Atomic/Core/Object.cpp#L326-L329).